docs/configuration.rst

```diff
@@ -14,24 +14,30 @@ Running in production
 Recommended settings
 --------------------
 
-Most default setting values in the application code base are suitable for production.
+Most default setting values in the application code base are suitable
+for production.
 
-But the set of settings mentionned below might deserve some review or adjustments:
+Once `PostgreSQL is installed <postgresql-install>`_, the settings about
+backends as shown in :file:`config/kinto.ini` can be uncommented in order
+to use *PostgreSQL*.
 
+Also, the set of settings mentionned below might deserve some review or adjustments:
 
 .. code-block :: ini
 
+    kinto.flush_endpoint_enabled = false
```

Comment:
Yes it is, but I believe it is better to review that the value is not set to true on prod :)